helm/flux-app/templates/policy-exception.yaml

```diff
@@ -0,0 +1,49 @@
+{{ if .Capabilities.APIVersions.Has "kyverno.io/v2alpha1/PolicyException" }}
+apiVersion: kyverno.io/v2alpha1
+kind: PolicyException
+metadata:
+  name: flux-app
+  namespace: "{{ .Release.Namespace }}"
+spec:
+  background: true
+  exceptions:
+  - policyName: disallow-host-path
+    ruleNames:
+    - host-path
+    - autogen-host-path
+    - autogen-cronjob-host-path
+  - policyName: require-run-as-nonroot
+    ruleNames:
+    - run-as-non-root
+    - autogen-run-as-non-root
+    - autogen-cronjob-run-as-non-root
+  - policyName: restrict-volume-types
+    ruleNames:
+    - restricted-volumes
```

Comment:
The ClusterPolicy `restricted-volumes` adds a restriction for `projected` volumes however this has been discovered to be required by the CRD installer job https://github.com/giantswarm/flux-app/pull/224

Do we expect this to collide or is there an override for this?

Reply:
This PolicyException takes that into account. We can start tightening security around flux-app as next step. For now, we allow all kinds of volumes, including `projected`.

Reply:
See https://github.com/giantswarm/flux-app/pull/225/commits/b3d147dc5f20371ed641845e4286c72882de8b9d